modules/marker_control/postinitClient.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+
+GVAR(Markers) = [];
+
+#include "settings.sqf"
+
+{
+  private _side = (_x select 0);
+  private _marker = (_x select 1);
+  private _playerSide = (side player);
+
+  if (_side != _playerSide) then {
+    _marker setMarkerAlphaLocal 0;
+  };
+} forEach GVAR(Markers);
```

Comment:
`apply` > `forEach` when the index is unused

Reply:
Fixed.